src/screens/Accounts/Accounts.js

```diff
@@ -254,61 +218,43 @@ class AccountsScreen extends React.Component<Props, State> {
   wallets(activeNetwork?: BlockchainNetwork): WalletsListItem[] {
     const {
       accounts,
-      navigation,
       balances,
       rates,
       baseFiatCurrency,
       theme,
     } = this.props;
 
-    const { smartWalletIcon, keyWalletIcon } = images(theme);
+    const { smartWalletIcon } = images(theme);
 
     const fiatCurrency = baseFiatCurrency || defaultFiatCurrency;
 
     const isEthereumActive = !!activeNetwork && (
       activeNetwork.id === BLOCKCHAIN_NETWORK_TYPES.ETHEREUM
     );
 
-    const wallets = accounts.map((account: Account): WalletsListItem => {
-      const { id, isActive, type } = account;
-      const accountBalances: Balances = balances[id];
-      const isActiveWallet = !!isActive && isEthereumActive;
-      const isSmartWallet = type === ACCOUNT_TYPES.SMART_WALLET;
-      let walletBalance;
-      if (accountBalances) {
-        const thisAccountBalance = calculateBalanceInFiat(rates, accountBalances, fiatCurrency);
-        walletBalance = formatFiat(thisAccountBalance, baseFiatCurrency);
-      }
-      return {
-        id: `ACCOUNT_${id}`,
-        type: 'ACCOUNT',
-        title: getAccountName(type),
-        balance: walletBalance,
-        isInitialised: true,
-        mainAction: () => this.switchWallet(account),
-        initialiseAction: null,
-        isActive: isActiveWallet,
-        iconSource: isSmartWallet ? smartWalletIcon : keyWalletIcon,
-        isSmartWallet,
-      };
-    });
-
-    if (!userHasSmartWallet(accounts)) {
-      wallets.push({
-        id: NEW_SMART_WALLET,
-        type: NEW_SMART_WALLET,
-        title: getAccountName(ACCOUNT_TYPES.SMART_WALLET),
-        balance: null,
-        isInitialised: false,
-        mainAction: null,
-        initialiseAction: () => { navigation.navigate(SMART_WALLET_INTRO); },
-        isActive: false,
-        iconSource: smartWalletIcon,
-        isSmartWallet: true,
+    return accounts
```

Comment:
idk, added filtering just because of migration and the rest was there, but from my opinion we need to check if any similar pattern is used elsewhere and then combine to single method, otherwise I don't think it's necessary, wdyt?